libsawtooth/src/journal/publisher/mod.rs

```diff
@@ -132,3 +131,88 @@ impl Clone for Box<dyn SyncPublisher> {
         self.box_clone()
     }
 }
+
+/// Tracks the state of the block that's being built
+struct CandidateBlock {
+    /// If `Some`, the maximum number of batches that are allowed in the block. If `None`, there is
+    /// no limit.
+    max_batches_per_block: Option<usize>,
+    /// Used to verify that batch signers are authorized to make transactions
+    permission_verifier: PermissionVerifier,
+    /// The block that's being built on top of
+    previous_block: BlockPair,
+    /// A reference to the previous block that's tracked by the block manager. This reference is
+    /// dropped when the candidate block is dropped (on finalization).
+    _previous_block_reference: BlockRef,
```

Comment:
Can you add a comment about why this is unused (underscore), took me a second to understand.

Reply:
Added more info